src/main/java/de/ellpeck/rockbottom/apiimpl/Renderer.java

```diff
@@ -414,24 +414,28 @@ public int getFlushes() {
 
     @Override
     public void renderSlotInGui(IGameInstance game, IAssetManager manager, ItemInstance slot, float x, float y, float scale, boolean hovered, boolean canPlaceInto) {
-        ITexture texture = manager.getTexture(SLOT_NAME);
-
-        int color = game.getSettings().guiColor;
+        renderSlotInGui(game, manager, slot, x, y, scale, hovered, canPlaceInto, true, -1);
+    }
 
-        if (!canPlaceInto) {
-            color = Colors.multiply(color, 0.5F);
-        } else if (!hovered) {
-            color = Colors.multiply(color, 0.75F);
+    @Override
+    public void renderSlotInGui(IGameInstance game, IAssetManager manager, ItemInstance slot, float x, float y, float scale, boolean hovered, boolean canPlaceInto, boolean renderBackground, int colorOverride) {
+        ITexture texture = manager.getTexture(SLOT_NAME);
+        if (renderBackground) {
+            if (colorOverride == -1) {
```

Comment:
-1 is still a color, setting that as a check wouldn't made sense :P Probably make a boolean to ask if there is an override